client/blocks/post-item/index.jsx

```diff
@@ -171,14 +182,10 @@ class PostItem extends React.Component {
 				<Card compact className={ cardClasses }>
 					<div className="post-item__detail">
 						<div className={ titleMetaClasses }>
-							{ isSiteVisible && (
-								<div className="post-item__site">
-									<SiteIcon size={ 16 } site={ site } />
-									<div className="post-item__site-title">
-										{ siteTitle }
-									</div>
-								</div>
-							) }
+							<div className="post-item__info">
+								<PostTypeSiteInfo globalId={ globalId } />
+								<PostTypePostAuthor globalId={ globalId } />
+							</div>
```

Comment:
We already have variables here called `isAuthorVisible` and `isSiteVisible` (better: `isSiteInfoVisible`).  What about using these variables to control whether these components are rendered here?  This should allow us to remove the dependency on these same parameters in the sub-components, for example [here](https://github.com/Automattic/wp-calypso/pull/17549/files#diff-a382ca2105ae121973f7a063fdea4857R19).

Reply:
Good point, I just pushed a commit to change that!